footsoldiers/src/footsoldiers.lisp

```diff
@@ -714,7 +717,9 @@
 (defun start-game (bot-relative-paths logging-config 
                    &key (current-directory nil)
                      (game-config *default-game-config*)
-                     (game-map *default-game-map*))
+                     (game-map *default-game-map*)
+                     (player1-error-stream *standard-output*)
```

Comment:
Shouldn't these default to standard error?

Reply:
Yes :)